### PR TITLE
docs: add color mode docs

### PR DESCRIPTION
**Pull request by cheton**
# Box

image

# Color Mode

To enable color mode, wrap your application in a `ColorModeProvider`, and specify the value prop with a color mode: `dark` or `light`.

```jsx disabled
import React from 'react';
import { ThemeProvider, ColorModeProvider } from '@trendmicro/react-styled-core';

function Example({ children }) {
  return (
    <ThemeProvider>
      <ColorModeProvider value="dark">
        {children}
      </ColorModeProvider>
    </ThemeProvider>
  );
}
```

Then you can use the `useColorMode` hook to get current color mode or change the color mode.

image

```jsx
function Example() {
  const { colorMode, toggleColorMode } = useColorMode();
  return (
    <Button onClick={toggleColorMode} alignItems="initial">
      {colorMode === 'light' ? (
        <>
          <TMIcon name="moon" />
          <Space width="2x" />
          Toggle Dark Mode
        </>
      ) : (
        <>
          <TMIcon name="sun" />
          <Space width="2x" />
          Toggle Light Mode
        </>
      )}
    </Button>
  );
}
```

To force a specific color mode, wrap your component in `LightMode` or `DarkMode` to override the global color mode.

image

```jsx
function Example() {
  const { colorMode, toggleColorMode } = useColorMode();

  return (
    <Stack direction="row" spacing="2x" shouldWrapChildren>
      <LightMode>
        <Button variantColor="blue">
          Light Mode Always
        </Button>
      </LightMode>
      <DarkMode>
        <Button variantColor="blue">
          Dark Mode Always
        </Button>
      </DarkMode>
      <Button variantColor="blue" onClick={toggleColorMode}>
        Toggle Mode
      </Button>
    </Stack>
  );
}
```

### CODE REVIEW

**Comment by cheton** [comment]
Changed `#faca2a` to `#faba2a` (206e45c00c5a64b29b4bdea2d4825eba3df4c3f9) for `yellow:50`

image

File: packages/styled-docs/pages/formcontrol.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# FormControl
```
Comment (austinhung81) [inline packages/styled-docs/pages/formcontrol.mdx]: Is there any FormControl component that we build?

Comment (cheton) [reply packages/styled-docs/pages/formcontrol.mdx]: This file should be in another PR. Thank you for your notice.

Comment (cheton) [reply packages/styled-docs/pages/formcontrol.mdx]: Removed in 1b44e38be4c986fcfa6d54acb29a6467d3fea971

**Review by tinaClin — APPROVED** [review]

**Review by austinhung81 — APPROVED** [review]

**Pull request merged**